app/boards/shields/pancake/pancake.keymap

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+#include <dt-bindings/zmk/outputs.h>
+
+#define DEF 0
+#define LWR 1
+#define RSE 2
+#define FNC 3
+
+/ {
+    keymap {
+        compatible = "zmk,keymap";
+
+        default_layer {
+            bindings = <
+                &kp ESC     &kp Q       &kp W       &kp E       &kp R       &kp T       &kp Y       &kp U       &kp I     &kp O     &kp P       &kp BSPC  
+                &kp TAB     &kp A       &kp S       &kp D       &kp F       &kp G       &kp H       &kp J       &kp K     &kp L     &kp SQT     &kp SEMI
+                &kp LSHFT   &kp Z       &kp X       &kp C       &kp V       &kp B       &kp N       &kp M       &kp COMMA &kp DOT   &kp UP      &kp ENTER
+                &kp LCTRL   &kp LALT    &kp LGUI    &mo FNC     &mo LWR     &kp SPACE   &kp SPACE   &mo RSE     &kp SLASH   &kp LEFT  &kp DOWN  &kp RIGHT
+            >;
+        };
+
+        lower_layer {
```

Comment:
Usually for these layers, you'd use `&trans` instead of `&none` as the default. It makes it easier to change the base layer and have some areas properly "fall through" without having to remember to change the upper layers also.

Reply:
keymap updated to use `&trans` instead.